mix.exs

```diff
@@ -66,11 +66,12 @@ defmodule Supavisor.MixProject do
       # pooller
       # {:poolboy, "~> 1.5.2"},
       {:poolboy, git: "https://github.com/abc3/poolboy.git", tag: "v0.0.2"},
-      {:partisan, git: "https://github.com/lasp-lang/partisan.git", tag: "v5.0.0-rc.12"},
+      {:partisan, git: "https://github.com/lasp-lang/partisan.git", tag: "v5.0.0-rc.13"},
       {:syn, "~> 3.3"},
       {:pgo, "~> 0.13"},
-      {:rustler, "~> 0.29.1"}
+      {:rustler, "~> 0.29.1"},
       # TODO: add ranch deps
+      {:ranch, "~> 2.0", override: true}
```

Comment:
are you sure that it's safe? I want to double check

Reply:
Also https://github.com/elixir-plug/plug_cowboy/pull/64